File: server/dotnet/HostedTokenization/HostedTokenizationService.cs
using OnlinePayments.Sdk.Domain;
using OnlinePayments.Sdk.Merchant;

namespace OnlinePayments.Sdk.Example;

public class HostedTokenizationService
{
    private readonly ILogger<HostedTokenizationService> _logger;

    private readonly IMerchantClient _merchantClient;

    public HostedTokenizationService(
        ILogger<HostedTokenizationService> logger,
        MerchantClientConfig merchantClientConfig
    )
    {
        _logger = logger;
        _merchantClient = merchantClientConfig.GetMerchantClient();
    }

    /// <summary>
    /// Initialize hosted tokenization
    /// </summary>
    /// <returns>Hosted tokenization response</returns>
    public async Task<CreateHostedTokenizationResponse> InitHostedTokenization() {
        _logger.LogInformation("Initializing hosted tokenization ...");
        var hostedTokenizationResponse = await _merchantClient.HostedTokenization.CreateHostedTokenization(new CreateHostedTokenizationRequest());
        _logger.LogInformation("Successful initialization for hosted tokenization - Hosted Tokenization Url: {}", hostedTokenizationResponse.HostedTokenizationUrl);
        return hostedTokenizationResponse;
    }
}

File: server/php/src/HostedTokenization/HostedTokenizationService.php
<?php

namespace MyApp\HostedTokenization;
use OnlinePayments\Sdk\Domain\CreateHostedTokenizationRequest;
use OnlinePayments\Sdk\Domain\CreateHostedTokenizationResponse;
use OnlinePayments\Sdk\Merchant\MerchantClientInterface;

/**
 * Hosted tokenization service
 */
class HostedTokenizationService
{
    private $merchantClient;

    public function __construct(MerchantClientInterface $merchantClient) {
        $this->merchantClient = $merchantClient;
    }

    public function initHostedTokenization(): CreateHostedTokenizationResponse {
        $hostedTokenizationResponse = $this->merchantClient->hostedTokenization()->createHostedTokenization(new CreateHostedTokenizationRequest());
        return $hostedTokenizationResponse;
    }
}